.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,4 +11,5 @@
  - [ ] tagged `@mapbox/studio` and/or `@mapbox/map-design-team` if this PR includes style spec or visual changes
  - [ ] tagged `@mapbox/gl-native` if this PR includes shader changes or needs a native port
  - [ ] apply changelog label ('bug', 'feature', 'docs', etc) or use the label 'skip changelog'
- - [ ] `<changelog>...</changelog>`
+ - [ ] write the changelog entry in the PR title - if more space is needed write between the `<changelog>` tags included below
+ - [ ] `<changelog></changelog>`
```

Comment:
What about making these two points a single checkbox, for example:

```suggestion
 - [ ] Add an entry inside this element for inclusion in the `mapbox-gl-js` release notes:`<changelog></changelog>`
```

Reply:
I originally had that in the PR that introduced this section. Last time it was removed during review, though I find the past conversation a little confusing: https://github.com/mapbox/mapbox-gl-js/pull/9023#discussion_r350462962

I like your suggestion just want to make sure the original concern isn't missed. @asheemmamoowala do you remember your concern?

Reply:
I'm 👍 on a single single entry.

Hard to remember now, but in https://github.com/mapbox/mapbox-gl-js/pull/9023#discussion_r350462962 I was thinking the checklist would not be for content - just for ✔️s , and the changelog text would be added elsewhere. Lets take that as a separate issue to update the template to make it easier to use and result in more consistent PR content.